connectivity.tf

```diff
@@ -20,3 +20,45 @@ resource "aws_vpc_peering_connection" "bgdc_to_internal_compute" {
     },
   )
 }
+
+data "aws_route_table" "bgdc_private" {
+  subnet_id = tolist(data.aws_subnet_ids.bgdc_private.ids)[0]
+}
+
+resource "aws_route" "bgdc_to_internal_compute" {
+  count                     = length(data.terraform_remote_state.internal_compute.outputs.pdm_subnet.ids)
+  route_table_id            = data.aws_route_table.bgdc_private.id
+  destination_cidr_block    = data.terraform_remote_state.internal_compute.outputs.pdm_subnet.cidr_blocks[count.index]
+  vpc_peering_connection_id = aws_vpc_peering_connection.bgdc_to_internal_compute.id
+}
+
+data "aws_route_table" "pdm" {
+  subnet_id = data.terraform_remote_state.internal_compute.outputs.pdm_subnet.ids[0]
+}
+
+resource "aws_route" "pdm_to_bgdc" {
+  count                     = length(tolist(data.aws_subnet_ids.bgdc_private.ids))
+  route_table_id            = data.aws_route_table.pdm.id
+  destination_cidr_block    = data.aws_subnet.bgdc_private[count.index].cidr_block
+  vpc_peering_connection_id = aws_vpc_peering_connection.bgdc_to_internal_compute.id
+}
+
+data "aws_network_acls" "bgdc_private" {
+  vpc_id = data.aws_vpc.bgdc.id
+
+  filter {
+    name   = "tag:Name"
+    values = ["bgdc-edc-vpc-stackPrivateNetworkAcl", ]
+  }
+}
+
+resource "aws_network_acl_rule" "bgdc_to_hive_metastore" {
```

Comment:
We have a policy/approach of not using Network ACLs anywhere. Why does BGDC think it needs them/why aren't SGs enough?

Reply:
This will be refactored (i.e. NACL's set to allow all) when we convert VPC creation Cloudformation into TF. Until then just injecting a required NACL rule